dunelegacy.appdata.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be submitted upstream.

Reply:
https://sourceforge.net/p/dunelegacy/patches/8/ done, but I don't see interest in upstream picking it up.